cpp/modmesh/toggle/pymod/wrap_Toggle.cpp

```diff
@@ -220,6 +279,145 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapToggle
 
 }; /* end class WrapToggle */
 
+namespace detail
+{
+
+struct Toggle2Python
+{
+
+    explicit Toggle2Python(Toggle & toggle)
+        : m_toggle(toggle)
+    {
+    }
+
+    static pybind11::object from_toggle(Toggle & toggle, std::string const & type)
+    {
+        pybind11::object r;
+        if (type == "solid")
+        {
+            Toggle2Python const o(toggle);
+            r = o.load_solid();
+        }
+        else if (type == "fixed")
+        {
+            Toggle2Python const o(toggle);
+            r = o.load_fixed();
+        }
+        else if (type == "dynamic")
+        {
+            Toggle2Python o(toggle);
+            pybind11::dict d = o.load_dynamic();
+            return d;
+        }
+        else
+        {
+            Toggle2Python o(toggle);
+            r = o.load();
+        }
+        return r;
+    }
+
+    pybind11::list load()
+    {
+        namespace py = pybind11;
+        py::list ret;
+        {
+            py::dict d;
+            d["fixed"] = load_fixed();
+            ret.append(d);
+        }
+        {
+            py::dict d;
+            d["dynamic"] = load_dynamic();
+            ret.append(d);
+        }
+        return ret;
+    }
+
+    pybind11::dict load_solid() const
+    {
+        namespace py = pybind11;
+        SolidToggle const & solid = m_toggle.solid();
+        py::dict ret;
+        ret["use_pyside"] = solid.use_pyside();
+        return ret;
+    }
+
+    pybind11::dict load_fixed() const
+    {
+        namespace py = pybind11;
+        FixedToggle const & fixed = m_toggle.fixed();
+        py::dict ret;
+        ret["show_axis"] = fixed.get_show_axis();
+        return ret;
+    }
+
+    pybind11::dict load_dynamic()
+    {
+        namespace py = pybind11;
+        DynamicToggleTable & table = m_toggle.dynamic();
+        std::vector<std::string> const keys = table.keys();
+        HierarchicalToggleAccess access(table);
+        py::dict ret;
+
+        for (auto const & fk : keys)
```

Comment:
The nested loop is not very hard to maintain, but not perfect either.  I'll live with it for a while.